pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -95,12 +100,19 @@ func Newk8sOrchestrator(ctx context.Context, controllerClusterFlavor cnstypes.Cn
 
 			k8sOrchestratorInstance = &K8sOrchestrator{}
 			k8sOrchestratorInstance.clusterFlavor = controllerClusterFlavor
+			k8sOrchestratorInstance.k8sClient = k8sClient
```

Comment:
It's used in GetVolumeAnnotations & UpdateVolumeAnnotations implementations while fetching and updating PVC annotations.
```
pvcObj, err := c.k8sClient.CoreV1().PersistentVolumeClaims(pvcNamespace).Get(ctx, pvcName, metav1.GetOptions{})
```

It can be removed if we use pvcLister() in informer.

Reply:
k8sClient is still needed while updating PVC annotations in `UpdateVolumeAnnotations` as the update operation will hit the api server.
```		
_, err = c.k8sClient.CoreV1().PersistentVolumeClaims(pvcNamespace).Update(ctx, pvcObj, metav1.UpdateOptions{})
```